### PR TITLE
Faster right clicks

### PR DESCRIPTION
**Pull request by mybluecorners**
## About the PR 
I was told about this via zewaka:
http://www.byond.com/forum/post/1326139?page=2#comment8198716
basically verbs on /mob are far less laggy than verbs on /obj. I reduced a lot of them. There is still a lot more that *could* be reduced for super fast right clicks, but I cleared all the easy ones.

I also reduced the density of admin right click menus because it turns out each admin option that is hidden from players causes players to lag more.

## Changelog

```
(u)mbc:
(*)Reduced lag when you right click on objects. You shouldn't notice any significant changes to the menu functions, but I did move around and merge some commands.
```

### CODE REVIEW

**Comment by ZeWaka** [comment]
Thanks to Clusterfack from /tg/station and other folks from over there for [solving this](https://github.com/tgstation/tgstation/pull/4516) back in 2014!

File: code/modules/admin/debug.dm [hunk 1]
```diff
@@ -735,22 +734,22 @@ body
 	T += "</table>"
 	usr.Browse(T, "window=browse_reagents;size=800x400")
 
-/client/proc/debug_check_possible_reactions(var/atom/O as mob|obj|turf)
+/atom/proc/debug_check_possible_reactions()
```
Comment (ZeWaka) [inline code/modules/admin/debug.dm:737]: Can't this be on every movable?

File: code/modules/status_system/statusSystem.dm [hunk 2]
```diff
@@ -1030,20 +1030,30 @@ var/list/statusGroupLimits = list("Food"=4)
 		duration = INFINITE_STATUS
 		maxDuration = null
 		var/mob/living/carbon/human/H
+		var/sleepcount = 5
```
Comment (ZeWaka) [inline code/modules/status_system/statusSystem.dm:1033]: Unrelated feature?

Comment (mybluecorners) [reply code/modules/status_system/statusSystem.dm:1033]: sleepcount is related - i removed the sleep-in verb from beds and instead added a timer where you lie in a bed for a few seconds you will automatically start sleeping

although now that you point this out that could be a bad idea. i'll think of some other way to do it later

**Comment by ZeWaka** [comment]
DNM on request of MBC re: buckle/sleeping thing and potentially forcing sleep

**Comment by ZeWaka** [comment]
If you want to fix travis failing, you'll have to pull from master.

**Comment by mybluecorners** [comment]
probably ready for merge now

**Comment by ZeWaka** [comment]
Needs an entry in the admin changelog for ~ and `.

**Pull request merged**